spec/in_paralell_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rspec'
 require_relative('../lib/in_parallel')
 include InParallel
-TMP_FILE = '/tmp/test_file.txt'
+TMP_FILE = 'tmp/test_file.txt'
```

Comment:
Unfortunately I can't...  There isn't a way to pass the reference of the tmp file back to the main process, and once the child process ends, the tmp file automatically gets deleted, which could cause the test to fail if the file gets created and deleted fast enough that it doesn't find it while polling for it's existence.